### PR TITLE
Adding test cases for invalid CREATE TABLE and column name

### PR DESCRIPTION
**Pull request by ahmed-shameem**
### **Description**

Adding test cases for invalid CREATE TABLE and column name
 
### **Issues Resolved**

[BABEL-1953]

### **Tests scenarios covered**

- Negative test scenarios

    o	Tests added for table creation without a table name
    o	Tests added for table creation without datatype of a column
    o	Tests added for table creation without column name but with datatypes
    o	Tests added for invalid table name
    o	Tests added for invalid column name
    o	Tests added for table without column
    o	Tests added for views which include invalid view name

Signed-off-by: Shameem Ahmed <shameemahmed20apr2000@gmail.com>

### CODE REVIEW

**Review by surendravishnoi — DISMISSED** [review]

File: test/JDBC/expected/BABEL-1953-vu-cleanup.out [hunk 1]
```diff
@@ -0,0 +1,5 @@
+-- Drop table created for VIEW
+DROP TABLE BABEL_1953_vu_prepare_t17
+GO
+
```
Comment (kuntalghosh) [inline test/JDBC/expected/BABEL-1953-vu-cleanup.out]: Please remove the extra newlines.

File: test/JDBC/expected/BABEL-1953-vu-prepare.out [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+~~ERROR (Message: syntax error near '@' at line 1 and character position 12)~~
+
+CREATE VIEW $ AS SELECT * FROM BABEL_1953_vu_prepare_t17
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: syntax error near '$' at line 1 and character position 12)~~
+
+
+
```
Comment (kuntalghosh) [inline test/JDBC/expected/BABEL-1953-vu-prepare.out]: Extra newlines..

File: test/JDBC/input/BABEL-1953-vu-prepare.sql [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+GO
+CREATE VIEW 123view AS SELECT * FROM BABEL_1953_vu_prepare_t17
+GO
+CREATE VIEW @ AS SELECT * FROM BABEL_1953_vu_prepare_t17
+GO
+CREATE VIEW $ AS SELECT * FROM BABEL_1953_vu_prepare_t17
+GO
+
+
+
```
Comment (kuntalghosh) [inline test/JDBC/input/BABEL-1953-vu-prepare.sql]: Extra newlines...

File: test/JDBC/upgrade/latest/schedule [hunk 4]
```diff
@@ -198,3 +199,4 @@ sys-all_sql_modules-dep
 sys-triggers
 sys-triggers-dep
 BABEL-2877
+
```
Comment (kuntalghosh) [inline test/JDBC/upgrade/latest/schedule]: Extra newline..

File: test/JDBC/expected/BABEL-1953-vu-cleanup.out (hunk 1 above)
Comment (ahmed-shameem) [reply test/JDBC/expected/BABEL-1953-vu-cleanup.out]: Addressed in the updated PR

File: test/JDBC/expected/BABEL-1953-vu-prepare.out (hunk 2 above)
Comment (ahmed-shameem) [reply test/JDBC/expected/BABEL-1953-vu-prepare.out]: Addressed in the updated PR

File: test/JDBC/input/BABEL-1953-vu-prepare.sql (hunk 3 above)
Comment (ahmed-shameem) [reply test/JDBC/input/BABEL-1953-vu-prepare.sql]: Addressed in the updated PR

File: test/JDBC/upgrade/latest/schedule (hunk 4 above)
Comment (ahmed-shameem) [reply test/JDBC/upgrade/latest/schedule]: Addressed in the updated PR

**Review by Deepesh125 — APPROVED** [review]

**Pull request merged**